kernel/virtio/solo5.lds

```diff
@@ -3,19 +3,12 @@ ENTRY(_start)
 SECTIONS {
     . = 0x100000;
 
-    /* Multiboot header */
-    .bootstrap :
-    {
-        *(.bootstrap)
-    }
-
-    . = ALIGN(0x1000);
-
     /* Code */
     _stext = .;
 
     .text :
     {
+        *(.data.multiboot)
```

Comment:
That's correct. The requirements are that the header be within the first 8k of the executable and be 4-byte aligned, both of which are taken care of. I tested GCE with these commits and everything works.